packages/verifiable-credentials/test/config.ts

```diff
@@ -54,3 +73,8 @@ export const connect = async (privateKey: string, customContexName?: string): Pr
     });
     return context;
 };
+
+
+export const base64Encoded = (data: string) => {
```

Comment:
Should be `base64encode()`

Reply:
This is a mistake from me it was supposed to be `base64dencoded()` because i initially encoded the `veridaUrl`, but i think its not need anymore